### PR TITLE
Add NanoDESI, NanoPOTS, multiple MS and LC-MS assays

### PR DESCRIPTION
**Pull request by jswelling**
This adds new assays: NanoDESI and its image pyramid, NanoPOTS (with no image pyramid), and the normal, Bottom Up, and Top Down variants of LC-MS and MS.  I believe none of these types contain personally identifying information.  The older equivalent LC-MS assays are *not* removed because we have not yet come to consensus on how to reclassify the existing datasets, so old and new LC-MS versions will exist together in the database until that consensus is reached.  

A block of comments preserving older MALDI positive and negative definitions was removed, since that change has passed testing.

### CODE REVIEW

**Comment by mccalluc** [comment]
Removing myself from the reviewers list: What really needs to be checked here is the content, and that's not an area where I need to be involved. Also, if responsibility is spread across too many people, everyone may think that someone else is giving it a close reading.

**Comment by yuanzhou** [comment]
@jswelling I looked at the PROD neo4j, currently we still have lots of datasets with the following types:
- MALDI-IMS-neg
- MALDI-IMS-pos

There is no match for 
- ALDI-IMS-neg_pyramid
- MALDI-IMS-pos_pyramid

The tests passed with commenting out these 4 types, how should we handle the existing datasets with the above two types? Also include @shirey to this discussion.

**Comment by jswelling** [comment]
@mccalluc the thing I want to bring to your attention is that this PR introduces the new MS and LC-MS assay types without removing the old types, which include existing datasets ultimately needing reclassification as new types.  I've proposed this so that we can begin to ingest new datasets in the interim.  I am concerned that this is something you will feel strongly about.

**Comment by yuanzhou** [comment]
@jswelling thanks for explaining the details.

**Review by yuanzhou — APPROVED** [review]
I'll merge this PR and redeploy on TEST once all other reviewers have approved.

**Review by cebriggs7135 — APPROVED** [review]

**Review by ngehlenborg — APPROVED** [review]

**Review by pdblood — APPROVED** [review]

**Pull request merged**
